far/dialog.hpp

```diff
@@ -216,6 +216,7 @@ class Dialog: public window
 	void* GetDialogData() const {return DataDialog;}
 	void InitDialog();
 	void Process();
+	void ProcessKey(int Key, size_t ItemPos);
```

Comment:
Not required for the variant with `std::optional<Manager::Key>`. Need to revert if we go with it.